test/artifact/artifact_installation_test.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "artifact_helper"
+
+class TestArtifactInstallation < Minitest::Test
+  parallelize_me!
+
+  def test_inspec_exists_linux
+    skip if windows?
+    stdout, stderr, status = Open3.capture3("/usr/bin/which inspec")
+    refute_match(/no inspec/, stdout)
+    assert_empty stderr
+    assert status
+  end
+
+  def test_inspec_exists_windows
+    skip unless windows?
+    stdout, stderr, status = Open3.capture3("Get-Command inspec")
+    assert_match(/inspec.bat/, stdout)
+    assert_empty stderr
+    assert status
```

Comment:
I think these should be merged and the conditional inside determines what command and regexp to use.